src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -248,5 +248,4 @@ OBA:2020005	lysine in blood amount	CHEBI:25094	lysine	PATO:0000070	amount	UBERON
 OBA:2060174		CHEBI:145810	insulin	PATO:0000070	amount	UBERON:0000178	blood
 OBA:2060175		CHEBI:35143	hemoglobin	PATO:0000070	amount	UBERON:0000178	blood
 OBA:2060176		CHEBI:25107	magnesium atom	PATO:0000070	amount	UBERON:0001088	urine
-OBA:2060177             PR:000022172    aspartate aminotransferase      PATO:0000070    amount  UBERON:0000178  blood
```

Comment:
This row is removed, but is it added somewhere else? It should at least be obsoleted?

Reply:
It is a duplicate of an equivalent existing term causing QC error. OBA:2060177 has not been included in any public release.
```
entity_attribute_location.tsv
198:OBA:VT0000203       blood aspartate transaminase amount     PR:000008153    aspartate aminotransferase, cytoplasmic PATO:0000070    amount  UBERON:0000178  blood
251:OBA:2060177         PR:000008153    aspartate aminotransferase, cytoplasmic PATO:0000070    amount  UBERON:0000178  blood
 ```

Reply:
OK